app/src/main/java/org/luxoft/sdl_core/MainActivity.java

```diff
@@ -63,13 +68,21 @@ public void onClick(View view) {
                     start_sdl_button.setEnabled(false);
                     stop_sdl_button.setEnabled(true);
                 }
+
+                if (isBleSupported() && isBluetoothPermissionGranted()) {
+                    startService(new Intent(MainActivity.this, org.luxoft.sdl_core.BleCentralService.class));
```

Comment:
@ychernysheva I think `org.luxoft.sdl_core.` prefix is redundant as these classes are in the same namespace

Reply:
Done in 49c191b0